rdrf/rdrf/settings.py

```diff
@@ -189,12 +188,13 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'django.middleware.security.SecurityMiddleware',
     'django_user_agents.middleware.UserAgentMiddleware',
-    'csp.middleware.CSPMiddleware',
-)
+]
 
 if SYSTEM_ROLE == SystemRoles.CIC_PROMS:
     MIDDLEWARE = PROMS_MIDDLEWARE
```

Comment:
default to csp enabled for proms , false for non-proms

Reply:
Done this.